spec.bs

```diff
@@ -280,9 +280,16 @@ To <dfn>look up penultimate redemption</dfn> given an [=/origin=] |issuer| and a
 1. If [=redemptionTimes=][(|issuer|,|topLevel|)] [=map/exists=], let |penultimateRedemption| be the |penultimateRedemption| field of the tuple [=redemptionTimes=][(|issuer|,|topLevel|)].
 1. Return |penultimateRedemption|.
 
-To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer| and an [=/origin=] |topLevel|, run the following steps:
+An user agent has <dfn>pstKeyCommitments</dfn>, a [=map=] where the keys are an [=/origin=] |issuer|, and the values are a [=map=] |commitment|. Populating |pstKeyCommitments| is [=implementation-defined=].
```

Comment:
You don't need to specify the names of map keys/values, i.e. you can drop `issuer` and `commitment`, but you need to specify what the second map (`commitment`) contains.

Reply:
nicki_21_21@hotmail.com

Reply:
Added a link to the definition above, not sure if there's extra text that needs to exist since that's a JSON structure vs IDL?

Reply:
Hmm yeah that's an interesting case. The way that you did this in the latest version seems fine. :)